packages/amplify-category-auth/resources/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -818,7 +832,7 @@ Resources:
     DependsOn: OAuthCustomResourceLogPolicy
   <% } %>
 
-  <%if (props.mfaConfiguration != 'OFF') { %>
+  <%if (!props.useEnabledMfas && props.mfaConfiguration != 'OFF') { %>
```

Comment:
is it simple to convert double negation into isMFAConfigurationOn ?

Reply:
The MFAConfiguration can either be 'OPTIONAL or ON' that's why the !=